backend/utils.ts

```diff
@@ -23,8 +19,30 @@ export const getTaxJarConfig = async (
       fromCity: channelSettings?.shipFromCity || "",
       fromStreet: channelSettings?.shipFromStreet || "",
     },
-    apiKey: channelSettings.apiKey,
+    apiKey: channelSettings?.apiKey || "",
     sandbox: channelSettings?.sandbox || true,
+    active: channelSettings?.active || false,
   };
   return taxJarConfig;
 };
+
+export const taxJarConfigIsValidToUse = (taxJarConfig: TaxJarConfig) => {
+  let message = "";
```

Comment:
Any idea if I can do this better? Assuming that in the future I could have more checks here (`if`s)